### ISSUE TITLE
Count not working

### ISSUE THREAD

**Issue by vdkdamian** [netreconlab/Parse-Swift#60]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/netreconlab/Parse-Swift/security/policy).
- [x] I have searched through [existing issues](https://github.com/netreconlab/Parse-Swift/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and the [Parse Swift SDK](https://github.com/netreconlab/Parse-Swift/releases). 

### Issue Description
I have this simple code that should return a count of objects, but it always returns 0.
When I use find(), I get the results as expected.

```
let shopsQuery = Shop.query()
return try await shopsQuery.count()
```
This returns 0.

```
let shopsQuery = Shop.query()
return try await shopsQuery.find()
```
This returns 13 objects

### Steps to reproduce
Use the asynchronous count method.

### Actual Outcome
0

### Expected Outcome
13

### Environment

Client
- Parse Swift SDK version: `4.16.2`
- Xcode version: `14.2`
- Operating system (iOS, macOS, watchOS, etc.): `iOS, macOS, ipadOS`
- Operating system version: `16.2, 13.1, 16.2`

Server
- Parse Server version: `5.2.3`
- Operating system: `Back4App`
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc): `AWS`

Database
- System (MongoDB or Postgres): `MongoDB`
- Database version: `3.6`
- Local or remote host (MongoDB Atlas, mLab, AWS, Azure, Google Cloud, etc): `Back4App`

**Comment by cbaker6** [comment]
Thanks for reporting! The fix will be released in `5.0.0-beta.8`. I recommend updating to that version even though it's a beta. There's one big feature I'm waiting to finish before releasing 5.0.0. Be sure to look at the change log for breaking changes.

**Comment by vdkdamian** [comment]
Great! Looking forward for the release!
